src/diffusers/models/attention_processor.py

```diff
@@ -4541,7 +4554,235 @@ def __call__(
 
         return hidden_states
 
+class IPAdapterXFormersAttnProcessor(torch.nn.Module):
```

Comment:
```suggestion
class IPAdapterXFormersAttnProcessor2_0(torch.nn.Module):
```

this corresponds to  IPAdapterAttnProcessor2_0, right?

Reply:
> this corresponds to IPAdapterAttnProcessor2_0, right?

About the class name, yes, it will replace 2.0 when Xformers is enabled, however, I kept that name because it was the naming pattern I found in the other existing classes. There is no Xformers class that ends with 2_0, unless there is a different situation that pairs with the IPAdapterAttnProcessor class, so we would have to have two Xformers classes for the IPAdapter, but since XFormers is already the differentiation itself, I believe there is no such need, am I correct?

Reply:
ok then!